djangoplicity/archives/contrib/search/forms.py

```diff
@@ -120,7 +120,6 @@ class AdvancedSearchForm (forms.Form):
 
         # take fields from options class, and sort them correctly
         fields = [(field_name, obj) for field_name, obj in options.AdvancedSearch.__dict__.items() if isinstance(obj, formsFields.Field)]
-        fields.sort(lambda x, y: cmp(x[1].creation_counter, y[1].creation_counter))
 
```

Comment:
Th "sort" line should be here

`fields.sort(key=lambda field: field[1].creation_counter)`

But I get an Attribute error even though the class should have the attribute (Inheritance) :(

![image](https://user-images.githubusercontent.com/36459449/102914729-bcc38480-444e-11eb-9610-257c8c5045d6.png)

I have tried many ways but I can't figure out why 😣

Reply:
What's creation_counter?. Did you check the ImageSizeSearchField?

Reply:
Yes, I did it. I checked the ImageSizeSearchField class in order to review the inheritance members:

ImageSizeSearchField inherits from SearchField, SearchField inherits from Field (django). Field has the "creation_counter" attribute, but I don't understand why ImageSearchField doesn't have this attribute? :(

I checked the builder functions and definitions and each is correct

I will check again today!